lib/grape/api.rb

```diff
@@ -68,15 +69,6 @@ def call(*args, &block)
         instance_for_rack.call(*args, &block)
       end
 
-      # Allows an API to itself be inheritable:
-      def make_inheritable(api)
```

Comment:
Why this change? Couldn't we just call `super` in `#inherited` and leave the rest as it is? :beers:

Reply:
Calling this method with `super` was creating warnings about method redefinations as `override_all_methods! ` gets called twice.

I guess `make_inheritable ` was defined in order to pass extra argument, instead of `super`.